README.md

```diff
@@ -38,7 +38,8 @@
 
 # 📖 [your_project_name] <a name="about-project"></a>
 
-> This is my portfolio. My first project in Microverse
+> This is my portfolio. My first project in Microverse. This website include all the projects i have worked on and still working on.
+ Thos portfolio website gives you a sneak peak of what  i can help you build.
 
 **[Portfolio setup and mobile first]** 
 
```

Comment:
- [ ] Kindly ignore the highlighted code. This comment has to do with the logo of this README.md file. Kindly remove the Microverse logo. If you have your logo, you can use it in place of it if not, kindly remove the Microverse logo. 👍